test/unit/node/update.test.ts

```diff
@@ -218,8 +218,8 @@ describe("update", () => {
   it("should reject if response has status code 500", async () => {
     if (isAddressInfo(_address)) {
       const mockURL = `http://${_address.address}:${_address.port}/reject-status-code`
-      let provider = new UpdateProvider(mockURL, settings())
-      let update = await provider.getUpdate(true)
+      const provider = new UpdateProvider(mockURL, settings())
```

Comment:
Very strange...I didn't make this change but when I ran `yarn lint` or `yarn fmt` locally, this changed happened?

Reply:
Huh I think we are passing `--fix` to auto-fix but I did not know they would change `let` to `const`.

Reply:
I didn't either! Maybe a new thing? Or maybe I did something janky...no idea

Reply:
Could be something new!  Either way, const looks right since we are not changing these values.